docs/DeveloperGuide.adoc

```diff
@@ -610,7 +610,10 @@ describes the behaviour of `DeleteTransactionCommand` at each step:
 
 === Repeaters
 
-// tag::repeateradd[]
```

Comment:
Please revert the changes you have made here for the repeater section. The "commented" tags are necessary for creating PPP

Reply:
Aah, I'll change that